### PR TITLE
Doc improvement

### PR DESCRIPTION
**Pull request by rookiegao**
Doc update

### CODE REVIEW

**Comment by rookiegao** [comment]
@leonardBang Can you check it for me？

File: docs/content/connectors/oracle-cdc.md [hunk 1]
```diff
@@ -205,8 +205,8 @@ Flink SQL> CREATE TABLE products (
      'port' = '1521',
      'username' = 'flinkuser',
      'password' = 'flinkpw',
-     'database-name' = 'XE',
-     'schema-name' = 'inventory',
+     'database-name' = 'XE',  -- XE is oracle SID
+     'schema-name' = 'inventory',  -- inventory is oracle user
```
Comment (leonardBang) [inline docs/content/connectors/oracle-cdc.md:209]: dabase-name could accept non SID value, schema=name caould accept non user value

File: docs/content/connectors/mysql-cdc.md [hunk 2]
```diff
@@ -491,7 +493,7 @@ public class MySqlSourceExample {
     MySqlSource<String> mySqlSource = MySqlSource.<String>builder()
         .hostname("yourHostname")
         .port(yourPort)
-        .databaseList("yourDatabaseName") // set captured database
+        .databaseList("yourDatabaseName") // set captured database, If you need to synchronize the whole database, Please set tableList to "".
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc.md:496]: ".*" is more recommended

**Review by leonardBang — COMMENTED** [review]
Thanks @rookiegao for the contribution, I left some comments

File: docs/content/connectors/oracle-cdc.md (hunk 1 above)
Comment (rookiegao) [reply docs/content/connectors/oracle-cdc.md:209]: I have some confusion.

I found the following answer in a link.

https://dba.stackexchange.com/questions/37012/difference-between-database-vs-user-vs-schema

Oracle's definitions of user and schema seem to be the same. Of course, this is limited to Oracle, which is not the case for other databases.

File: docs/content/connectors/mysql-cdc.md (hunk 2 above)
Comment (rookiegao) [reply docs/content/connectors/mysql-cdc.md:496]: I will improve it

File: docs/content/connectors/oracle-cdc.md (hunk 1 above)
Comment (leonardBang) [reply docs/content/connectors/oracle-cdc.md:209]: ping @luoyuxia could you also help have a check ?

Comment (luoyuxia) [reply docs/content/connectors/oracle-cdc.md:209]: @rookiegao Hi, so is it possible that user 'SCOTT‘ create a table `inventory.t1'`? Then it seems the `inventory` will be the `schema-name`.

Comment (rookiegao) [reply docs/content/connectors/oracle-cdc.md:209]: I see. I'll cancel the modification

**Comment by rookiegao** [comment]
@leonardBang @luoyuxia can you guys help me review it ?

**Comment by luoyuxia** [comment]
lgtm.

File: docs/content/connectors/mysql-cdc.md [hunk 3]
```diff
@@ -31,6 +31,8 @@ In order to setup the MySQL CDC connector, the following table provides dependen
 
 Download [flink-sql-connector-mysql-cdc-2.3-SNAPSHOT.jar](https://repo1.maven.org/maven2/com/ververica/flink-sql-connector-mysql-cdc/2.3-SNAPSHOT/flink-sql-connector-mysql-cdc-2.3-SNAPSHOT.jar) and put it under `<FLINK_HOME>/lib/`.
 
+**Note:** XXX snapshot version is the code corresponding to the development branch. Users need to download the source code and compile the corresponding jar. Users should use the released version, such as flink-sql-connector-mysql-cdc-2.2.0.jar, the released version will be available in the Maven central warehouse.
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc.md:34]: Could you also add this note for other connectors? other parts LGTM

Comment (rookiegao) [reply docs/content/connectors/mysql-cdc.md]: done

**Review by leonardBang — APPROVED** [review]
Thanks @rookiegao for the update, LGTM

**Comment by leonardBang** [comment]
@rookiegao Could you also open a backport PR for branch `release-2.2` ？

**Pull request merged**